entity-management-web/src/test/java/eu/europeana/entitymanagement/web/EMControllerTest.java

```diff
@@ -52,5 +55,23 @@ public void registerEntity_concept() throws Exception {
 //	assertEquals(1, res);
 	
     }
+    
+    @Test
+    public void disableEntity() throws Exception {
+    	MultiValueMap<String, String> params = new LinkedMultiValueMap<>();
+        params.add(CommonApiConstants.PARAM_WSKEY, "apidemo");
+        String typePathParam = "Timespan";
+        String namespacePathParam = "base";
+        String identifierPathParam = "3";
+
+
+        String result = mockMvc
+        		.perform(MockMvcRequestBuilders.delete(BASE_SERVICE_URL+"{type}/{namespace}/{identifier}",typePathParam,namespacePathParam,identifierPathParam).params(params).header(HttpHeaders.AUTHORIZATION, "enable_authorization")
+        		.header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
+        		.andExpect(status().isAccepted()).andReturn().getResponse().getHeader("info:");
+	
+        System.out.println(result);
```

Comment:
unit/intergration test must ensure that the operation was actually performed in the API. In this case we need to send a request using the retrieve method and check the status of the response to be 410

Reply:
Please see the comment below. It is addressed there.